partner_contact_sale_info_propagation/models/res_partner.py

```diff
@@ -1,50 +1,64 @@
 # Copyright 2019 Tecnativa - Ernesto Tejeda
+# Copyright 2021 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from odoo import api, models
 from lxml import etree
+from odoo.tools import config
 
 
 class ResPartner(models.Model):
     _inherit = "res.partner"
 
+    def _check_propagation_allowed(self):
+        return bool(
+            not config["test_enable"]
+            or (
+                config["test_enable"]
+                and not self.env.context.get("prevent_propagation", False)
```

Comment:
This should be the exact contrary:
```suggestion
                and not self.env.context.get("test_propagation")
```
and only enable such context on this module tests.